etc/profile-a-l/dino-im.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for dino-im
+# Description: Modern XMPP Chat Client using GTK+/Vala, Ubuntu specific bin name
+# This file is overwritten after every install/update
+##quiet
+# Persistent local customizations
+include dino-im.local
+# Persistent global definitions
+include globals.local
```

Comment:
We should avoid including globals.local twice, so it needs:
# added by included profile
#include globals.local